### ISSUE TITLE
Audio files are not saved: cannot unpack non-iterable NoneType object

### ISSUE THREAD

**Issue by OlegRuban-ai** [yihong0618/SunoSongsCreator#21]
Error when trying to generate audio. The profile shows that the files have been created and the links during processing are correct, but saving does not occur.
![1](https://github.com/yihong0618/SunoSongsCreator/assets/61088564/7dd8f95b-7c87-42d3-babc-b05f42b3a8f9)

**Comment by OlegRuban-ai** [comment]
There is also a problem with saving files without the is_custom mode. None appears in the file name

Version: 0.5.0
![1](https://github.com/yihong0618/SunoSongsCreator/assets/61088564/65c113f7-5c87-4dc4-bec7-5a10b317c290)

**Comment by yihong0618** [comment]
> There is also a problem with saving files without the is_custom mode. None appears in the file name
> 
> Version: 0.5.0 ![1](https://private-user-images.githubusercontent.com/61088564/321309354-d36de463-47e5-48c8-bf61-421dec47515f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3ODM5NDAsIm5iZiI6MTcxMjc4MzY0MCwicGF0aCI6Ii82MTA4ODU2NC8zMjEzMDkzNTQtZDM2ZGU0NjMtNDdlNS00OGM4LWJmNjEtNDIxZGVjNDc1MTVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDIxMTQwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjNzBiYTY3Y2EyNGMzNzM5Y2NhZDBiOGQ4ZTdjNmUzODIyOTJhZTM4M2Q2MzVmMDUzNmVjMDA3Yjk5YzNmNzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4iE2G_tvMzUBv8xJQ4S1P2MqN77oaevsNG2uqmpGLCM)

this is suno problem it generate `None` link

**Comment by yihong0618** [comment]
can you try again?

**Comment by OlegRuban-ai** [comment]
Yep, it's problem from Suno website. I don't understand why they stopped generate songs on ukrainian or russian languages. When i start generate Suno gives error whith message "prompt likely maolformed". I send message to support and wait answer.
